op-deployer/pkg/deployer/standard/standard.go

```diff
@@ -279,6 +281,17 @@ func L1ProxyAdminOwner(chainID uint64) (common.Address, error) {
 	}
 }
 
+func ProtocolVersionsOwner(chainID uint64) (common.Address, error) {
+	switch chainID {
+	case 1:
+		return common.Address(validation.StandardConfigRolesMainnet.ProtocolVersionsOwner), nil
+	case 11155111:
+		return common.Address(validation.StandardConfigRolesSepolia.ProtocolVersionsOwner), nil
+	default:
+		return common.Address{}, fmt.Errorf("unsupported chain ID: %d", chainID)
+	}
+}
+
```

Comment:
Should we go ahead and use this same pattern for other standard values so we don't have the same value hardcoded two places?

* `GuardianAddressFor`
* `ChallengerAddressFor`
* `SuperchainProxyAdminAddrFor`
* `L1ProxyAdminOwner`

Reply:
updated all except the superchain proxy admin addr - that one doesn't have an equivalent in the roles file